crux-core/src/crux/api/alpha/TransactionOperation.java

```diff
@@ -1,11 +1,26 @@
 package crux.api.alpha;
 
+import clojure.lang.Keyword;
 import clojure.lang.PersistentVector;
 
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import static crux.api.alpha.CruxId.cruxId;
+import static crux.api.alpha.Document.document;
+import static crux.api.alpha.Util.keyword;
+
 public abstract class TransactionOperation {
+    static final Keyword TX_PUT = keyword("crux.tx/put");
+    static final Keyword TX_EVICT = keyword("crux.tx/evict");
+    static final Keyword TX_CAS = keyword("crux.tx/cas");
+    static final Keyword TX_DELETE = keyword("crux.tx/delete");
+
     TransactionOperation() {
     }
 
+    // May want to move these - are inherited by the classes
```

Comment:
the static constructors?

Reply:
Yep, in the generated Javadocs they're listed as inherited functions